capstone/src/main/webapp/index.html

```diff
@@ -7,6 +7,12 @@
     <script src="script.js"></script>
   </head>
   <body>
-    hello
+    <h1>Explore Clubs</h1>
+    <div id="content">
+      <div id="club-listings">
+          
+      </div>
```

Comment:
If you aren't adding anything in the div tag, could you remove the blank line and put the closing div tag on the same line as the opening div tag? I think it will make it look cleaner